task1/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "task1",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
are @testing-library/* packages from CRA?? 

and the main questions why are they as dependencies? not devDep? 🧐

Reply:
They are from CRA, I didn't install them manually.